pom.xml

```diff
@@ -1888,7 +1888,8 @@
                     <overWriteSnapshots>false</overWriteSnapshots>
                     <overWriteIfNewer>true</overWriteIfNewer>
                     <excludeGroupIds>org.apache.hadoop,org.apache.hbase,org.apache.hive,com.google.protobuf,\
-                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson</excludeGroupIds>
+                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson,\
+                      org.apache.tez</excludeGroupIds>
```

Comment:
We have a direct dependency on tez-api, defined in the root pom and in cdap-explore/pom.xml.